querydsl-sql/pom.xml

```diff
@@ -35,6 +35,7 @@
       <groupId>joda-time</groupId>
       <artifactId>joda-time</artifactId>
       <version>1.6</version>
+      <optional>true</optional>
```

Comment:
Do you see the possibility to bump the version of `joda-time` as well? In the issue some users also report version conflicts.

Reply:
Sure, if the QueryDSL team approves it